.github/workflows/additional-ci-image-checks.yml

```diff
@@ -89,42 +89,40 @@ jobs:
   # delay cache refresh. It does not attempt to upgrade to newer dependencies.
   # We only push CI cache as PROD cache usually does not gain as much from fresh cache because
   # it uses prepared airflow and provider packages that invalidate the cache anyway most of the time
-  # push-early-buildx-cache-to-github-registry:
-  #   name: Push Early Image Cache
-  #   uses: ./.github/workflows/push-image-cache.yml
-  #   permissions:
-  #     contents: read
-  #     # This write is only given here for `push` events from "apache/airflow" repo. It is not given for PRs
-  #     # from forks. This is to prevent malicious PRs from creating images in the "apache/airflow" repo.
-  #     # For regular build for PRS this "build-prod-images" workflow will be skipped anyway by the
-  #     # "in-workflow-build" condition
-  #     packages: write
-  #   secrets: inherit
-  #   with:
-  #     runs-on: ${{ inputs.runs-on }}
-  #     cache-type: "Early"
-  #     include-prod-images: "false"
-  #     push-latest-images: "false"
-  #     image-tag: ${{ inputs.image-tag }}
-  #     python-versions: ${{ inputs.python-versions }}
-  #     branch: ${{ inputs.branch }}
-  #     use-uv: "true"
-  #     include-success-outputs: ${{ inputs.include-success-outputs }}
-  #     constraints-branch: ${{ inputs.constraints-branch }}
-  #     docker-cache: ${{ inputs.docker-cache }}
-  #   if: inputs.canary-run == 'true' && inputs.branch == 'main'
+  push-early-buildx-cache-to-github-registry:
+    name: Push Early Image Cache
+    uses: ./.github/workflows/push-image-cache.yml
+    permissions:
+      contents: read
+      # This write is only given here for `push` events from "apache/airflow" repo. It is not given for PRs
+      # from forks. This is to prevent malicious PRs from creating images in the "apache/airflow" repo.
+      # For regular build for PRS this "build-prod-images" workflow will be skipped anyway by the
+      # "in-workflow-build" condition
+      packages: write
+    secrets: inherit
+    with:
+      # Runs on Public runners
+      cache-type: "Early"
+      include-prod-images: "false"
+      push-latest-images: "false"
+      platform: "linux/amd64"
+      python-versions: ${{ inputs.python-versions }}
+      branch: ${{ inputs.branch }}
+      constraints-branch: ${{ inputs.constraints-branch }}
+      use-uv: "true"
+      include-success-outputs: ${{ inputs.include-success-outputs }}
+      docker-cache: ${{ inputs.docker-cache }}
+    if: inputs.canary-run == 'true' && inputs.branch == 'main'
 
   # Check that after earlier cache push, breeze command will build quickly
   check-that-image-builds-quickly:
-    timeout-minutes: 5
+    timeout-minutes: 11
```

Comment:
This job had not been tesitng what it was supposed to be testing. Having IMAGE_TAG, it was just pulling the image rarther than building it using cache and timeout of 2 minutes was far too low for building it. We might tweak it in the future as well when the cache gets stabilized.